airflow/__init__.py

```diff
@@ -85,15 +85,15 @@
     "version": (".version", "", False),
     # Deprecated lazy imports
     "AirflowException": (".exceptions", "AirflowException", True),
-    "Dataset": (".assets", "Dataset", True),
+    "Dataset": (".sdk.definitions.asset", "Dataset", True),
```

Comment:
We should also change the line for Asset right?

Reply:
I think we should keep it as `Dataset`. I think we're using the subclass `Dataset(Asset)` here